CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project are documented in this file.
 
+## 0.0.7 (2020-07-31)
```

Comment:
There are no strict rules on SemVer for pre-releases, but wouldn't it be better to increase the minor version?

Reply:
Changing the CRD from v1beta1 to v1 was a similar breaking change. I would not bump the version to 0.1.x as it would signal a stable release and we are not there yet.